### PR TITLE
Improve and extend constant usage

### PR DESCRIPTION
**Pull request by Gymnasiast**

### CODE REVIEW

File: src/openrct2/rct2.h [hunk 1]
```diff
@@ -32,6 +32,8 @@
 #define RCT2_MAX_VEHICLES_PER_RIDE_ENTRY    4
 #define RCT2_DOWNTIME_HISTORY_SIZE          8
 #define RCT2_CUSTOMER_HISTORY_SIZE          10
+#define RCT2_MAX_SPRITES                    10000
```
Comment (janisozaur) [inline src/openrct2/rct2.h]: What about https://github.com/OpenRCT2/OpenRCT2/blob/c6e162e31c9630edc221bc97cb72542b29176432/src/openrct2/world/sprite.h#L26 ?

Comment (Gymnasiast) [reply src/openrct2/rct2.h]: https://github.com/OpenRCT2/OpenRCT2/wiki/RCT12_MAX_SOMETHING-versus-MAX_SOMETHING

Surprised you'd ask this, as I have been doing this preparation for moving off SV6 for quite a while now.

Comment (janisozaur) [reply src/openrct2/rct2.h]: I know, but I usually saw it was migrating from `rct_x` to `x`, whereas here it is duplicated. Just verifying it's meant to be.

Comment (Gymnasiast) [reply src/openrct2/rct2.h]: It is, yes.

**Pull request merged**